### ISSUE TITLE
DeprecationWarning: `urllib.parse.splitquery()` is deprecated as of 3.8

### ISSUE THREAD

**Issue by cclauss** [internetarchive/openlibrary#6851]
### Evidence / Screenshot (if possible)
When we run pytest in our GitHub Actions, we get the following warning
```
=============================== warnings summary ===============================
openlibrary/coverstore/tests/test_doctests.py::test_doctest[openlibrary.coverstore.utils]
openlibrary/coverstore/tests/test_doctests.py::test_doctest[openlibrary.coverstore.utils]
openlibrary/coverstore/tests/test_doctests.py::test_doctest[openlibrary.coverstore.utils]
  /home/runner/work/openlibrary/openlibrary/openlibrary/coverstore/utils.py:88:
   DeprecationWarning: urllib.parse.splitquery() is deprecated as of 3.8, use urllib.parse.urlparse() instead
    base, query = splitquery(URL)
```
https://github.com/internetarchive/openlibrary/blob/master/openlibrary/coverstore/utils.py#L88

### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual: pytest warning
* Expected: No pytest warning

### Details

- **Logged in (Y/N)?**
- **Browser type/version?**
- **Operating system?**
- **Environment (prod/dev/local)?** prod

**Comment by scottbarnes** [comment]
I'd like to work on this.

**Comment by cclauss** [comment]
Deprecated in Py38 but is still available in Python 3.11 which is not yet released.
